newport/mockXPS.go

```diff
@@ -142,6 +152,7 @@ func (c *MockController) setPosition(axis string, pos float64) {
 
 // MoveAbs = public interface; moveTo = asynchronous internal interface
 func (c *MockController) moveTo(axis string, pos float64) {
+	c.stop[axis] = false
```

Comment:
Need to wrap access to the map with the mutex (`c.Lock(); c.stop[axis]=false; c.Unlock()`) as-is the code can panic from unsynchronized map writes